### PR TITLE
Section policies display

### PR DESCRIPTION
**Pull request by ViRuSTriNiTy**
see https://github.com/pkp/pkp-lib/issues/2638

### CODE REVIEW

File: classes/submission/form/SubmissionSubmitStep1Form.inc.php [hunk 1]
```diff
@@ -38,6 +38,20 @@ function fetch($request) {
...
+		$templateMgr->assign('sectionId', $request->getUserVar('sectionId'));
+
+		// Get section policies for this context
+		$sectionPolicies = array();
+		foreach ($sectionOptions as $sectionId => $sectionTitle) {
+			$section = $sectionDao->getById($sectionId);
+
+			$sectionPolicy = $section ? $section->getLocalizedPolicy() : null;
+			$sectionPolicyPlainText = trim(PKPString::html2text($sectionPolicy));
+			if (strlen($sectionPolicyPlainText) > 0)
```
Comment (asmecher) [inline classes/submission/form/SubmissionSubmitStep1Form.inc.php]: (See https://github.com/pkp/pkp-lib/pull/4135/files#r224624538)

File: locale/en_US/locale.xml [hunk 2]
```diff
@@ -101,6 +101,7 @@
 	<message key="section.title">Section title</message>
 	<message key="section.abbreviation">Abbreviation</message>
 	<message key="section.abbreviation.example">(For example, Articles=ART)</message>
+	<message key="section.policy">Section Policy</message>
```
Comment (asmecher) [inline locale/en_US/locale.xml]: Does this also require a PR for OMP?

File: templates/submission/form/section.tpl [hunk 3]
```diff
@@ -14,3 +14,43 @@
...
+
+{foreach from=$sectionPolicies key="sectionPolicySectionId" item="content"}
+	{assign var=sectionPolicyHidden value=false}
+	{if $sectionPolicySectionId != $sectionId}
+		{assign var=sectionPolicyHidden value=true}
+	{/if}
+	{include file="submission/form/sectionPolicy.tpl" sectionId=$sectionPolicySectionId content=$content hidden=$sectionPolicyHidden}
+{/foreach}
+
+<script type="text/javascript">
```
Comment (asmecher) [inline templates/submission/form/section.tpl]: We're between two Javascript frameworks -- our own homebrew one, which we're gradually working to replace, and a new vue.js-based one. The old framework discouraged Javascript code in template files, and the new one favours vue templates. Maybe @NateWr has a recommendation for this...?

File: classes/submission/form/SubmissionSubmitStep1Form.inc.php (hunk 1 above)
Comment (ViRuSTriNiTy) [reply classes/submission/form/SubmissionSubmitStep1Form.inc.php]: See my comments in the linked PR

File: templates/submission/form/section.tpl (hunk 3 above)
Comment (ViRuSTriNiTy) [reply templates/submission/form/section.tpl]: Yep, i definitely need some help here on how to implement the dynamic refresh without a javascript block. It is somewhat a quick and dirty solution.

File: locale/en_US/locale.xml (hunk 2 above)
Comment (ViRuSTriNiTy) [reply locale/en_US/locale.xml]: > Does this also require a PR for OMP?

I don't think so as the change is part of pkp/ojs and is used in `sectionPolicy.tpl` which is also part of pkp/ojs only.

File: templates/submission/form/section.tpl (hunk 3 above)
Comment (ViRuSTriNiTy) [reply templates/submission/form/section.tpl]: Not relevant anymore due to PR provided by @NateWr

**Pull request merged**